app/pages/lab-pages-editor/components/TasksPage.jsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-console */
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable react/require-default-props */
+/* eslint-disable radix */
+
+import { useWorkflowContext } from '../context.js';
+import strings from '../strings.json'; // TODO: move all text into strings
+
+import GripIcon from '../icons/GripIcon.jsx';
+
+export default function TasksPage() {
+  const { workflow, update } = useWorkflowContext();
+  const isActive = true; // TODO
+
+  function placeholderEventHandler() {
+    console.log('+++ TODO');
+  }
+
+  console.log('+++ workflow: ', workflow);
+
+  if (!workflow) return null;
+
+  return (
+    <div className="tasks-page">
+      <div className="workflow-title flex-row">
+        <h2 className="flex-item">{workflow?.display_name}</h2>
+        <span className="workflow-id">{`#${workflow?.id}`}</span>
+        {(isActive) ? <span className="status-active">Active</span> : <span className="status-inactive">Inactive</span>}
+      </div>
+      <section>
+        <h3>Tasks</h3>
+        <div className="flex-row">
+          <button
+            className="flex-item big primary"
+            onClick={placeholderEventHandler}
+            type="button"
+          >
+            Add a new Task +
+          </button>
+          {/* Dev observation: the <select> should have some label to indicate it's for choosing the starting task. */}
+          <select
+            aria-label="Choose starting test"
+            className="flex-item"
+          >
+            <option disabled={true}>Choose starting test</option>
+          </select>
+        </div>
+        <ul className="steps-list">
+          {/* WARNING: this should be workflow.steps */}
+          {Object.entries(workflow.tasks).map(([key, val]) => (
+            <StepItem
+              task={val}
+              taskKey={key}
+            />
+          ))}
+        </ul>
+      </section>
+    </div>
+  );
+}
+
+// WARNING/TODO: this should be handling steps, not tasks
+function StepItem({ task, taskKey }) {
+  if (!task || !taskKey) return null;
+
+  const text = task.instruction;
+
+  return (
+    <li className="step-item">
+      <div className="grip-bar flex-row spacing-bottom-XS">
+        <GripIcon color="#A6A7A9" />
+      </div>
```

Comment:
This should be a button.

Reply:
You could add a keyboard handler to this button, so that the cursor keys move items up and down in the list, when the drag handle is focused.

Reply:
Alternatively, make the whole `StepItem` a focusable element that responds to the cursor keys. That would be trickier, since each step item contains buttons and form inputs that also need to respond to the pointer and keyboard. 

An example of how that can go wrong is https://github.com/zooniverse/front-end-monorepo/issues/5422.